mico-core/src/main/java/io/github/ust/mico/core/resource/ApplicationResource.java

```diff
@@ -224,6 +229,39 @@
         return ResponseEntity.noContent().build();
     }
 
+    @ApiOperation(value = "Adds a new association between a MicoApplication and a KafkaFaasConnector (MicoService). " +
+        "Multiple instances of a KafkaFaasConnector are allowed per MicoApplication.")
+    @PostMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}/" + PATH_KAFKA_FAAS_CONNECTOR + "/{" + PATH_VARIABLE_KAFKA_FAAS_CONNECTOR_VERSION + "}")
```

Comment:
Using the same variable at the same position with different types and semantics seems like a really bad idea. Using post on the url `/<short-name>/<version>/kafka-faas-connector/` with a bode containing the version would be a cleaner solution to this.

Reply:
I agree with you regarding the problem.
With your solution it would be inconsistent with the existing `addServiceToApplication` method.
Another idea:
- Keep the POST method as it is but change the DELETE endpoint to:
  `/<short-name>/<version>/kafka-faas-connector/<kf-connector-version>/<instance-id>`

Reply:
Covered by #793